packages/contracts-bedrock/test/L1/L1StandardBridge.t.sol

```diff
@@ -426,27 +469,40 @@ contract L1StandardBridge_DepositETHTo_Test is PreBridgeETHTo {
     ///      EOA or contract can call depositETHTo.
     ///      ETH ends up in the optimismPortal.
     function test_depositETHTo_succeeds() external {
-        _preBridgeETHTo({ isLegacy: true });
+        _preBridgeETHTo({ isLegacy: true, value: 600 });
         l1StandardBridge.depositETHTo{ value: 600 }(bob, 60000, hex"dead");
         assertEq(address(optimismPortal).balance, 600);
     }
 }
 
+contract L1StandardBridge_DepositETHTo_TestFail is Bridge_Initializer {
+    /// @dev Tests that depositETHTo reverts with custom gas token.
+    function test_depositETHTo_customGasToken_reverts() external {
+        vm.mockCall(
+            address(systemConfig), abi.encodeWithSignature("gasPayingToken()"), abi.encode(address(1), uint8(2))
+        );
+        vm.expectRevert("StandardBridge: cannot bridge ETH with custom gas token");
+
+        l1StandardBridge.depositETHTo(address(0), 0, hex"");
+    }
+}
+
 contract L1StandardBridge_BridgeETHTo_Test is PreBridgeETHTo {
     /// @dev Tests that bridging ETH to a different address succeeds.
     ///      Emits ETHDepositInitiated and ETHBridgeInitiated events.
     ///      Calls depositTransaction on the OptimismPortal.
     ///      Only EOA can call bridgeETHTo.
     ///      ETH ends up in the optimismPortal.
     function test_bridgeETHTo_succeeds() external {
-        _preBridgeETHTo({ isLegacy: false });
+        _preBridgeETHTo({ isLegacy: false, value: 600 });
         l1StandardBridge.bridgeETHTo{ value: 600 }(bob, 60000, hex"dead");
         assertEq(address(optimismPortal).balance, 600);
     }
+}
 
-    /// @dev Tests that bridging ETH reverts if a custom gas token is used.
-    function test_bridgeETHTo_isCustomGasToken_reverts() external {
-        vm.prank(alice, alice);
+contract L1StandardBridge_BridgeETHTo_TestFail is PreBridgeETHTo {
+    /// @dev Tests that bridging reverts with custom gas token.
+    function test_bridgeETHTo_customGasToken_reverts() external {
         vm.mockCall(
             address(systemConfig), abi.encodeWithSignature("gasPayingToken()"), abi.encode(address(1), uint8(2))
```

Comment:
This one could be good to move to being a fuzz test

Reply:
I thought we'd just revert for all values with custom gas token? if so, I don't think it'd make sense to fuzz?